src/dramatiq_abort/backend.py

```diff
@@ -1,14 +1,14 @@
 import abc
-from typing import Iterable, List, Optional
+from typing import Any, Dict, Iterable, Optional, Tuple
 
 
 class EventBackend(abc.ABC):
     """ABC for event backends."""
 
     @abc.abstractmethod
     def wait_many(
-        self, keys: List[bytes], timeout: int
-    ) -> Optional[bytes]:  # pragma: no cover
+        self, keys: Iterable[str], timeout: int
+    ) -> Tuple[Optional[str], Optional[Dict[str, Any]]]:  # pragma: no cover
```

Comment:
I would rather use `NamedTuple` if the tuple components are not obvious. Also, Let's make the tuple optional, not its component.

```python
class Event(NamedTuple):
  type: str
  params: Dict[str, Any]

def wait_many(...) -> Optional[Event]: ...
def poll(...) -> Optional[Event]: ...
def notify(self, items: Iterable[Event], ...): ...
```
```